vendor/bat-native-ledger/src/bat/ledger/internal/endpoints/request_for.cc

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/endpoints/request_for.h"
+
+#include "net/http/http_status_code.h"
+#include "net/http/http_util.h"
+
+namespace ledger::endpoints {
+
+void SendImpl(LedgerImpl* ledger,
```

Comment:
This function is super dense. Is there a way to restructure it slightly so that it's easier to read? (For instance, by not doing recursion on `SendImpl` when there's no params and no need to call it again.) If not, that's fine too.